src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -407,7 +407,7 @@ const WorkflowView = _.flow(
           onDismiss: () => this.setState({ launching: false }),
           onSuccess: submissionId => {
             const { methodRepoMethod: { methodVersion, methodNamespace, methodName, methodPath, sourceRepo } } = modifiedConfig
-            Ajax().Metrics.captureEvent(Events.workflowLaunch, { ...extractWorkspaceDetails(workspace), methodVersion, sourceRepo, methodPath: sourceRepo === 'agora' ? `${methodNamespace}/${methodName}` : methodPath })
+            Ajax().Metrics.captureEvent(Events.workflowLaunch, { ...extractWorkspaceDetails(workspace), fromSnapshot: entitySelectionModel.type.description === 'process snapshot table', methodVersion, sourceRepo, methodPath: sourceRepo === 'agora' ? `${methodNamespace}/${methodName}` : methodPath })
```

Comment:
does `entitySelectionModel.type === processSnapshotTable` work here, instead of looking at the description? Just in case the description changes in the future.

Reply:
It's a symbol so reading directly from `entitySelectionModel.type` doesn't work. I agree reading the description is a bit clunky but I couldn't get anything else to read correctly, so if you have a better idea please let me know! I considered extracting an 'isSnapshot' property to state, but it's only used twice so that seemed like it might be overkill.

Reply:
I see `=== processSnapshotTable` elsewhere in this file - line 751 for example - does that not work in this context? I haven't tried it myself and am unsure, so this is an honest question.

Reply:
Yeah I also thought that would work! That's what I did first but it seemed like it wasn't getting updated correctly. It would work fine the first time I ran a workflow, but then it would never change afterwards, so it seemed like it wasn't updating when you changed the root entity type from the dropdown.

Reply:
I'm honestly not 100% sure why reading the description works but just evaluating it doesn't…

Reply:
Ok! Got Isaac to explain what symbols are and now it's fixed, I was comparing it to a string instead of to the symbol itself, which was why it didn't work before but does now!